execution_engine/src/core/engine_state/execute_request.rs

```diff
@@ -5,7 +5,7 @@ use casper_types::{ProtocolVersion, PublicKey, SecretKey};
 use super::deploy_item::DeployItem;
 use crate::shared::newtypes::Blake2bHash;
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
Is it necessary to `Clone` this object? It can be expensive to have multiple copies of it and we really should only need one.

Reply:
It's needed in benches, but isn't used in the node this way.

Reply:
Got it. Thanks.